recipe/meta.yaml

```diff
@@ -28,7 +28,12 @@ requirements:
   run:
     - beautifulsoup4
     - chardet
-    - conda >=4.13
+    - conda  >=4.13
+    - conda-index
+    # - conda-verify  >=3.0.2  # optional as of CB 3.12.0
+    - contextlib2   # [py<34]
+    - enum34        # [py<34]
+    - pathlib2      # [py<3]
```

Comment:
Are these lines really necessary? 😅 

I didn't know we had to support Python `<3.4`...

Reply:
No I didn't touch the recipe as part of this PR.